src/Component/NameField/NameField.tsx

```diff
@@ -33,13 +33,9 @@ import {
 
 import './NameField.less';
 
-// default props
-export interface NameFieldDefaultProps {
+export interface NameFieldProps {
```

Comment:
Could you add a migration note for this breaking change to the commit message? Something along the lines of: Interface `NameFieldDefaultProps` was removed. Use `NameFieldProps` instead.